test_submission_schema/v1.json

```diff
@@ -0,0 +1,168 @@
+                    "type": "string",
+                    "description": "Test Observer execution ID."
+                }
+            }
+        },
+        "results": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "required": [
+                    "id",
+                    "test_description",
+                    "certification_status",
+                    "category_id",
+                    "outcome",
+                    "comments",
+                    "io_log",
+                    "duration",
+                    "type",
+                    "template_id"
+                ],
+                "properties": {
+                    "id": {
+                        "type": "string",
+                        "pattern": "^[a-zA-Z0-9_]+\\.[a-zA-Z0-9_]+\\.[a-zA-Z0-9_]+::[a-zA-Z0-9_]+/[a-zA-Z0-9_]+$",
+                        "description": "The unique identifier of the test case."
+                    },
+                    "test_description": {
+                        "type": "string",
+                        "description": "The description of the test case.",
+                        "maxLength": 80
+                    },
+                    "certification_status": {
+                        "type": "string",
+                        "description": "The certification status of the test case.",
+                        "enum": [
+                            "blocker",
+                            "non-blocker"
+                        ]
+                    },
+                    "category_id": {
+                        "type": "string",
+                        "pattern": "^[a-zA-Z0-9_]+\\.[a-zA-Z0-9_]+\\.[a-zA-Z0-9_]+::[a-zA-Z0-9_]+$",
+                        "description": "The unique identifier of the test case category."
+                    },
+                    "outcome": {
+                        "type": "string",
+                        "description": "The outcome of the test case.",
+                        "enum": [
+                            "passed",
+                            "failed",
+                            "skipped"
+                        ]
+                    },
+                    "comments": {
+                        "type": "string",
+                        "description": "Any additional comments or observations about the test case."
+                    },
+                    "io_log": {
+                        "type": "string",
+                        "description": "The stdout and stderr for the test case."
+                    },
+                    "duration": {
+                        "type": "number",
+                        "minimum": 0,
+                        "description": "The duration of the test case in milliseconds."
+                    },
+                    "type": {
+                        "type": "string",
+                        "description": "The type of test case.",
+                        "enum": []
```

Comment:
We may want to limit the value of this field to a list of choices, is there a list of expected value for this?

Reply:
same question for `category_id`

Reply:
`category_id` should not be an enum, it's not up to us to define it here.

For type, @Hook25 we couldn't find any value except for `test`, any other possible value?

Reply:
This is what checkbox puts there `"type": "test",`, we can drop this?

Reply:
Sure